packages/rocketchat-ui-account/account/accountPreferences.html

```diff
@@ -139,6 +139,17 @@
 									<label><input type="radio" name="mergeChannels" value="-1" checked="{{checked 'mergeChannels' undefined true}}" /> {{_ "Default"}}</label>
 								</div>
 							</div>
+							<div class="input-line double-col" id="sendOnEnter">
+								<label>{{_ "Send_On_Enter"}}</label>
+								<div>
+									<select class="input-monitor" name="sendOnEnter">
+										<option value="1" selected="{{selected 'sendOnEnter' 1 true}}">{{_ "True"}}</option>
```

Comment:
Why using numbers instead of string like `both`, `none`, `desktop`, or `true`, `false`, `desktopOnly`?